documentation/CAMARA-API-access-and-user-consent.md

```diff
@@ -69,6 +69,24 @@ The Purpose definition (naming + description) and format within CAMARA follows t
 
 Purpose must be specified in the authorization request for a CAMARA Three-Legged Access Token, this is achieved by extending the `scope` parameter as defined in [Purpose as a Scope](CAMARA-Security-Interoperability.md#purpose-as-a-scope) section of the CAMARA Security and Interoperability Profile.
 
+## Controlling User-Interaction
+
+In CIBA User-Interaction in initiated by the API Producer if Consent is needed and NOT granted. The `login_hint` parameter is mandatory in CIBA and is used to identify the authentication devicde. Consent is captured out-of-band by e.g. sending a push notification, or by sending an SMS with a code or link, or by sending a message by some other means to the authentication devide. CAMARA does not mandate a communication method or any message format or content.
```

Comment:
It is not mandatory for the API provider to seek user or end-user consent. They can just reject the authentication request instead.

The CIBA `login_hint` parameter itself will not necessarily be being used by the **API consumer** to identify an authentication device. They may be using it solely to identify a subscription. If the API Provider then uses that parameter to identify and authentication device, that is up to them. But they might alternatively identify an e-mail address, for example.

It is important to understand that, for some APIs, it is user, and not end user, consent that is required, even if the end user is different from the user.

```suggestion
If the API Consumer is using CIBA, User-Interaction MAY be initiated by the API Producer if Consent is needed and has NOT been granted. The `login_hint` parameter is mandatory in CIBA, and MAY be used by the API Producer to identify an authentication device for out-of-band consent capture. Consent MAY be captured out-of-band by, for example, sending a push notification, or by sending an SMS with a code or link, or by sending a message by some other means to the identified authentication device. CAMARA does not mandate any communication method nor any message format nor any message content.
```

Reply:
Please consider my format suggestions in other comments...